xmtp_mls/src/client.rs

```diff
@@ -224,7 +225,7 @@ pub struct FindGroupParams {
     pub created_after_ns: Option<i64>,
     pub created_before_ns: Option<i64>,
     pub limit: Option<i64>,
-    pub include_dm_groups: bool,
+    pub conversation_type: Option<ConversationType>,
```

Comment:
I changed all these `include_dm` booleans to `conversation_type` so that we could have all 3 options
just dms, just groups, and both.